Nodes/ClassNode.cs

```diff
@@ -45,29 +45,32 @@ public IntPtr Address
 
 		public event NodeEventHandler NodesChanged;
 
-		internal ClassNode(bool notifyClassCreated)
+		internal ClassNode(bool notifyClassCreated, IntPtr Address = default(IntPtr))
 		{
 			Contract.Ensures(AddressFormula != null);
 
 			Uuid = new NodeUuid(true);
-
+			if (Address == IntPtr.Zero)
+			{
 #if WIN64
-			Address = (IntPtr)0x140000000;
+				this.Address = (IntPtr)0x140000000;
 #else
-			Address = (IntPtr)0x400000;
+				this.Address = (IntPtr)0x400000;
 #endif
-
+			}
+			else
+				this.Address = Address;
 			if (notifyClassCreated)
 			{
 				ClassCreated?.Invoke(this);
 			}
 		}
 
-		public static ClassNode Create()
+		public static ClassNode Create(IntPtr Address = default(IntPtr))
```

Comment:
Same here, lowercase and add a second method without a parameter which calls this method.